### ISSUE TITLE
[bash-completion] make it work with mingw packages

### ISSUE THREAD

**Issue by lazka** [msys2/MSYS2-packages#2438]
The following makes it work, maybe something we can set on start:

```
XDG_DATA_DIRS=$MINGW_PREFIX/share:/usr/local/share/:/usr/share/ bash -l
```

**Comment by 1480c1** [comment]
as a note, a lot of projects look for `pkgconfig --variable=completionsdir bash-completion` for the directory to install the bash completion scripts into

**Comment by lazka** [comment]
Fix in #2455

> as a note, a lot of projects look for `pkgconfig --variable=completionsdir bash-completion` for the directory to install the bash completion scripts into

That won't work for mingw packages I guess.

Nevertheless I think this is an improvement at least.
